Bump backbeat & zenko-operator

welansari opened the pull request:
Issue: ZENKO-4679

Commit:
bump backbeat & zenko-operator

Issue: ZENKO-4679

bert-e commented:
# Hello kerkesni,



My role is to assist you with the merge of this
pull request. Please type `@bert-e help` to get information
on this process, or consult the [user documentation](
https://bert-e.scality.net/scality/zenko/doc/user).





*Status report is not available.*

bert-e commented:
# Incorrect fix version



The `Fix Version/s` in issue ZENKO-4679 contains:


* *None*



Considering where you are trying to merge, I ignored possible hotfix versions and I expected to find:



* `2.6.39`

* `2.7.35`

* `2.8.14`


Please check the `Fix Version/s` of ZENKO-4679, or the target
branch of this pull request.

bert-e commented:
# Request integration branches



Waiting for integration branch creation to be requested by the user.

To request integration branches, please comment on this pull request with the following command:

```
/create_integration_branches
```

Alternatively, the `/approve` and `/create_pull_requests` commands will automatically
create the integration branches.

welansari commented:
/create_integration_branches

bert-e commented:
# Conflict




A conflict has been raised during the creation of
integration branch `w/2.7/bugfix/ZENKO-4679` with contents from `bugfix/ZENKO-4679`
and `development/2.7`.

**I have not created the integration branch.**


Here are the steps to resolve this conflict:

```bash
 $ git fetch
 $ git checkout -B w/2.7/bugfix/ZENKO-4679 origin/development/2.7
 $ git merge origin/bugfix/ZENKO-4679
 $ # <intense conflict resolution>
 $ git commit
 $ git push -u origin w/2.7/bugfix/ZENKO-4679
```





*The following options are set:* **create_integration_branches**

bert-e commented:
# Integration data created



I have created the integration data for the additional destination branches.

* this pull request will merge `bugfix/ZENKO-4679` into
`development/2.6`
* [w/2.7/bugfix/ZENKO-4679](https://github.com/scality/zenko/compare/development/2.7...w/2.7/bugfix/ZENKO-4679) will be merged into `development/2.7` 
* [w/2.8/bugfix/ZENKO-4679](https://github.com/scality/zenko/compare/development/2.8...w/2.8/bugfix/ZENKO-4679) will be merged into `development/2.8` 



The following branches will **NOT** be impacted:

* `development/2.5`




You can set option `create_pull_requests` if you need me to create
**integration pull requests** in addition to integration branches, with:

```
@bert-e create_pull_requests
```







*The following options are set:* **create_integration_branches**

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* 2 peers











*The following options are set:* **create_integration_branches**

Commit:
Fix incompatibility issue in ctst tests

aws4-axios 2.x only supports versions of
axios below 1.x, this causes an issue when
other packages install versions above that,
as aws4-axios does not specify that the version
should be below 1.x

Issue: ZENKO-4679

welansari commented:
@bert-e create_pull_requests

bert-e commented:
# Integration data created



I have created the integration data for the additional destination branches.

* this pull request will merge `bugfix/ZENKO-4679` into
`development/2.6`
* [w/2.7/bugfix/ZENKO-4679](https://github.com/scality/zenko/compare/development/2.7...w/2.7/bugfix/ZENKO-4679) will be merged into `development/2.7` (pull request #1978)
* [w/2.8/bugfix/ZENKO-4679](https://github.com/scality/zenko/compare/development/2.8...w/2.8/bugfix/ZENKO-4679) will be merged into `development/2.8` (pull request #1979)



The following branches will **NOT** be impacted:

* `development/2.5`




*Follow* integration pull requests if you would like to be notified of
build statuses by email.







*The following options are set:* **create_pull_requests, create_integration_branches**

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* 2 peers











*The following options are set:* **create_pull_requests, create_integration_branches**

welansari commented:
/approve

bert-e commented:
# In the queue



The changeset has received all authorizations and has been added to the
relevant queue(s). The queue(s) will be merged in the target development
branch(es) as soon as builds have passed.

The changeset will be merged in:


* :heavy_check_mark: `development/2.6`

* :heavy_check_mark: `development/2.7`

* :heavy_check_mark: `development/2.8`



The following branches will **NOT** be impacted:

* `development/2.5`



There is no action required on your side. You will be notified here once
the changeset has been merged. In the unlikely event that the changeset
fails permanently on the queue, a member of the admin team will
contact you to help resolve the matter.

*IMPORTANT*

Please do not attempt to modify this pull request.

* Any commit you add on the source branch will trigger a new cycle after the
  current queue is merged.
* Any commit you add on one of the integration branches will be **lost**.

If you need this pull request to be removed from the queue, please contact a
member of the admin team now.




*The following options are set:* **approve, create_pull_requests, create_integration_branches**

bert-e commented:
# 


I have successfully merged the changeset of this pull request
into targetted development branches:


* :heavy_check_mark: `development/2.6`

* :heavy_check_mark: `development/2.7`

* :heavy_check_mark: `development/2.8`



The following branches have **NOT** changed:

* `development/2.5`



Please check the status of the associated issue ZENKO-4679.

Goodbye kerkesni.
